.version

```diff
@@ -1 +1 @@
-4.2.0
\ No newline at end of file
+gould-4.2.0
```

Comment:
don't commit this version

Reply:
yes, had to leave this in for pending test for binance, will be removed in final commit

Reply:
Binance testing done, fixed.